.github/quarkus-ecosystem-issue.java

```diff
@@ -16,8 +16,9 @@
 
 //usr/bin/env jbang "$0" "$@" ; exit $?
 
-//DEPS org.kohsuke:github-api:1.321
-//DEPS info.picocli:picocli:4.2.0
+//JAVA 21
```

Comment:
That's not mandatory for the changes, but there is also no reason to support older versions.